packages/react-map-gl/esbuild.js

```diff
@@ -0,0 +1,10 @@
+import { build } from "../../esbuild.js";
+import { createRequire } from "module";
+
+const { resolve } = createRequire(import.meta.url);
+
+await build({
+  entryPoints: [resolve("react-map-gl")],
+  outfile: "./dist/index.js",
+  external: ["mapbox-gl", "react", "react-dom"]
```

Comment:
I think perhaps we need to do Eik import mapping in these cases.

The built file is doing a runtime require of `"react"`, which needs to be mapped to the URL for React on Eik. I can set it up.

Reply:
Do you know if anyone else is likely to depend on `mapbox-gl` directly? If not we can add it as a dependency and bundle it here. Otherwise we need to add it as a shared module same as `maplibre-gl`

Reply:
Did a quick search for other entries of `mapbox-gl` and there are some repos that have it as a direct dependency. So it might be wise to add it as a shared module